components/testimonial/Testimonial.module.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../styles/globals.scss';
+
+.container {
+	padding: $space-md;
+	width: 30%;
+	background: rgb(0, 0, 0);
+	background: linear-gradient(
```

Comment:
Why are we using a linear gradient with a solid color background here?  In the spec image the component is transparent, let's fix this to match the style in the spec.  See screenshot:
![Screen Shot 2022-05-07 at 5 06 00 PM](https://user-images.githubusercontent.com/58790413/167271929-51729c75-3a3c-4931-b239-1c6cde73bffd.png)